magento/pwa-studio
# [doc] release notes 9.0.0

## Description

- Creates release notes for PWA Studio 9.0.0
- Updates the compatibility table
- Updates the Top Community Contributors for this release

## Related Issue

Closes PWA-1235

## Acceptance

@davemacaulay 
@ericerway 
@awilcoxa 

### Verification Stakeholders

@davemacaulay 
@ericerway 
@awilcoxa 

### Specification



### Verification Steps

1. Open CHANGELOG
2. Verify accurate content

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
